src/routes/trip.route.js

```diff
@@ -178,4 +179,33 @@ router.patch(
   requestMiddleware.checkIfAlreadyChanged,
   tripController.updateTripRequestStatus
 );
+
+/**
+ * @swagger
+ *
+ * /trip/trip-requests/{tripRequestID}:
```

Comment:
Good work Eric but I don't think the route will be like this you have to remove that `/trip` if you are dealing with trip requests.